addons/account_peppol/views/res_config_settings_views.xml

```diff
@@ -29,6 +29,15 @@
                                             ]}">
                                     <field name="account_peppol_endpoint_warning"/>
                                 </div>
+                                <div class="alert alert-warning mt-3"
+                                     colspan="2"
+                                     role="alert"
+                                     attrs="{'invisible': ['|',
+                                                ('country_code', '!=', 'BE'),
+                                                ('account_peppol_eas', 'in', (False, '0208'))
```

Comment:
Maybe also add `('is_account_peppol_participant', '=', False),` ?

Reply:
yes, missed that! thanks :slightly_smiling_face: